samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi

```diff
@@ -0,0 +1,37 @@
+@App:name("ConnectSiddhiAppsSink")
```

Comment:
Rename as ConnectSiddhiAppsSink1 and ConnectSiddhiAppsSink2

Reply:
The requested changes are addressed by the commit 6d13673.